R/nested.R

```diff
@@ -41,12 +41,31 @@ compute_parse_data_nested <- function(text) {
 #' [utils::getParseData()] is used to obtain a flat parse table from `text`.
 #' @param text A character vector.
 #' @return A flat parse table
+#' @importFrom tidyr separate
```

Comment:
Actually no, I use extract now. But since the whole package is imported anyways in R/ws.R, we would not need tidyr imports anyways.